consensus/parlia/parlia.go

```diff
@@ -278,7 +278,7 @@ func New(
 		validatorSetABIBeforeLuban: vABIBeforeLuban,
 		validatorSetABI:            vABI,
 		slashABI:                   sABI,
-		signer:                     types.NewEIP155Signer(chainConfig.ChainID),
```

Comment:
maybe types.LatestSigner() is better as it can make choice from the ChainID.

Reply:
accpeted